wrappers/s2i/python-conda/requirements.txt

```diff
@@ -0,0 +1 @@
+seldon-core
```

Comment:
It would be good to pin these `requirements.txt` files to the latest version (`0.5.1` right now).